### PR TITLE
Add customTarget to TAP file

### PR DESCRIPTION
**Pull request by momanmalik**
resolves: #348
signed-off-by: Moman Malik <momanmalik@ibm.com>

### CODE REVIEW

**Comment by momanmalik** [comment]
Grinder: [Grinder/27471](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/27471/)

**Comment by momanmalik** [comment]
Tap results with no custom target.
Grinder: [Grinder/27475](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/27475)

File: scripts/resultsSum.pl [hunk 1]
```diff
@@ -247,6 +247,12 @@ sub resultReporter {
 		print $fhOut "#SHA.txt content: \n# " . $SHAContent . "\n";
 
 		print $fhOut "# Timestamp: " . $timeStamp . " UTC \n";
+
+		#add customTarget to TAP file
+  		if ($customTarget ne '') {
+  			print $fhOut "# custom target: " . "${customTarget}" . "\n";
```
Comment (llxia) [inline scripts/resultsSum.pl:253]: Can we change `custom target` to `CUSTOM_TARGET`, so it matches with the parameter name? Thanks

**Review by renfeiw — APPROVED** [review]
LGTM

**Pull request merged**